src/lib/Components/Bidding/Screens/ConfirmBid.tsx

```diff
@@ -64,14 +64,14 @@ const MAX_POLL_ATTEMPTS = 20
 const queryForBidPosition = (bidderPositionID: string) => {
```

Comment:
while we're here, thoughts on renaming to queryforBid**der**Position as a `BidderPosition` is a thing, but a `BidPosition` isn't a thing?